lib/taskbook.js

```diff
@@ -306,6 +307,15 @@ class Taskbook {
     render.successCreate(note);
   }
 
+  clipboard(ids) {
+    ids = this._validateIDs(ids);
+    const {_data} = this;
+    // Copy the first id
+    const id = ids[0];
+    clipboardy.writeSync(_data[id].description);
+    render.successClipboard(id);
+  }
```

Comment:
There is no reason to limit the coping process to only a single item description. Instead, we could modify our method to accept multiple target item ids as input, store each target's description in an array and then synchronously write each description, followed by the newline character, to the system's clipboard. This way, when pasted, the result will be a clear and readable stack of sentences.

```js
  copyToClipboard(ids) {
    ids = this._validateIDs(ids);
    const {_data} = this;
    const descriptions = [];

    ids.forEach(id => descriptions.push(_data[id].description));

    clipboardy.writeSync(descriptions.join('\n'));
    render.successCopyToClipboard(ids);
  }
```